pkg/cloudprovider/cloud.go

```diff
@@ -135,6 +135,7 @@ type Instances interface {
 	NodeAddressesByProviderID(ctx context.Context, providerID string) ([]v1.NodeAddress, error)
 	// InstanceID returns the cloud provider ID of the node with the specified NodeName.
 	// Note that if the instance does not exist or is no longer running, we must return ("", cloudprovider.InstanceNotFound)
```

Comment:
clarify the "is no longer running" text above. Possible wording: "...if the instance does not exist or is no longer running and will not start running again, ...".

@thockin might have thoughts on generic wording for the idea of "terminated" instances.

Reply:
I think changing it to:
```
// Note that if the instance does not exist, we must return ("", cloudprovider.InstanceNotFound)
```
in addition to the line below should be clear enough.

Reply:
@liggitt made ^ this change PTAL